Frontend/ui-customer/src/Screens/CustomerUtils/ReportIssueChatScreen/Handler.purs

```diff
@@ -285,11 +285,26 @@ callSupportHandler updatedState = do
 reOpenIssueHandler :: ReportIssueChatScreenState -> FlowBT String FlowState
 reOpenIssueHandler updatedState = do
   let language = fetchLanguage $ getLanguageLocale languageKey
-  let updateIssueReqBody = UpdateIssueReqBody {status : "OPEN"}
-  (UpdateIssueRes _) <- Remote.updateIssue (fromMaybe "" updatedState.data.issueId) language updateIssueReqBody
-  pure unit
-  modifyScreenState $ ReportIssueChatScreenStateType (\_ -> updatedState {props {isResolved = false}})
-  App.BackT $ App.NoBack <$> (pure $ IssueReportChatScreenFlow)
+      selectedOptionId = fromMaybe "" $ map (\option -> option.issueOptionId) updatedState.data.selectedOption
+      updateIssueReqBody = UpdateIssueReqBody { status : "REOPENED" }
+  (GetOptionsRes _) <- Remote.getOptionsBT language updatedState.data.categoryId selectedOptionId (fromMaybe "" updatedState.data.issueId)
```

Comment:
why we are ignoring the response of get API? if its not needed, we can remove right ?

Reply:
This Api will save the current selected option to chatId. If this api call is not made then this select option will be missing from chats when someone reloads it.